src/content/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-apm-agents.mdx

```diff
@@ -21,158 +21,1190 @@ redirects:
   - /docs/logs/log-monitoring/enable-logs/configure-logs-context-apm-agents
 ---
 
-Logs in context makes it easy to link to your log data with related data across the rest of the New Relic platform. For example, you can easily correlate log messages to a related error trace or distributed trace in [APM](/docs/apm/applications-menu/monitoring/apm-overview-page-view-transaction-apdex-usage-data). This is accomplished by appending trace IDs to the corresponding application logs, automatically filtering to those logs from the [errors](/docs/apm/applications-menu/error-analytics/error-analytics-explore-events-behind-errors) or [distributed trace](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data) UIs.
+When you need to correlate log data with other telemetry data, enable logs in context in New Relic. Logs in context adds metadata that links your log data with related data, like [error](/docs/apm/apm-ui-pages/error-analytics/errors-page-find-fix-verify-problems/) or [distributed trace](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data) information in New Relic APM.
+
+## See the root cause of issues across your platform [#compatibility-requirements]
+
+By bringing all of your application and infrastructure data together in a single solution, you can more quickly get to the root cause of issues. Logs in context help you quickly see meaningful patterns and trends.
+
+Don't spend extra time trying to narrow down all your logs from different parts of your platform. Instead, enable logs in context to see the exact log lines you need to identify and resolve a problem.
+
+## Basic process to enable logs in context [#enable-logs]
+
+The process to enable logs in context is basically the same, regardless of which APM agent you use to monitor your application:
+
+1. Update to a supported APM agent version for your app, and enable [distributed tracing](/docs/understand-dependencies/distributed-tracing/enable-configure/enable-distributed-tracing/).
+2. Configure logs in context for your APM agent, as explained in this document, or for your [infrastructure monitoring agent](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent/).
+3. Configure a [supported log forwarder](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/enable-log-management-new-relic/) to collect your application logs, as explained in the documentation for logs in context setup.
+4. View your logs within the context of your apps or infrastructure in [New Relic One](/docs/logs/log-management/ui-data/use-logs-ui/#links).
+
+The main differences in this procedure are which log forwarders are available for your agent, and how to configure the specific log forwarder.
```

Comment:
> The main differences in this procedure are which log forwarders are available for your agent, and how to configure the specific log forwarder

This is true, but I suspect not what you mean to say. How about this: (Although I'm fuzzy on what we call the specific language logs in context enrichment thingies)
"The main differences in this procedure are which language log appenders you use to enrich your log data, and how to configure that specific log appender"